### DISCUSSION TITLE
Welcome to Public-APIs Discussions!

### DISCUSSION THREAD

**Discussion by n0shake** [n0shake/Public-APIs#400] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by WebCraftPhil** [comment]
Hey my name's Phil I'm 33 years old. I'm just interested in how complex problems have been solved for the general public using api's. I've recently started using cloud platforms and I'm interested in what could be built to help my fellow humans. Also I'm broke so if I could earn some kind of money that would be nice as well. Knowledge is the way. Thank you.

**Comment by Stanwukong** [comment]
Hello everyone, my name is Femi, 25. I'm just looking for some inspiration.
